### PR TITLE
Portfolio Website: Complete Mobile Version

### PR DESCRIPTION
**Pull request by Shisui6**
## Create Footer and Contact Form

- Create and style footer element
- Create and style form element
- Configure form only to accept correct values
- Collect data by using the Formspree service

### CODE REVIEW

File: resources/CSS/style.css [hunk 1]
```diff
@@ -249,3 +250,87 @@ ul {
...
+  right: 0;
+  top: 10px;
+  width: 220px;
+  height: 709px;
+}
+
+form {
+  margin-top: 50px;
+}
+
```
Comment (tresorsawasawa) [inline resources/CSS/style.css:294]: - [ ] As required for the project, kindly make sure you use `Flexbox` to layout the `footer-content` or `footer elements` and your project will be completed successfully

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Shisui6 and @yonas44 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,160 +1,267 @@
...
+                <p>Skills</p>
+                <img src="./resources/images/right.svg" alt="right" />
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <footer>
+        <div class="footer-content">
+          <img src="./resources/images/footer.png" alt="footer" />
```
[the remark this answers is not in the document]
Comment (Shisui6) [reply index.html:225]: Hey @tresorsawasawa,

Thanks for your suggestion. The Figma design does not have a background image attached to it. It uses a background color and an image to implement the design. This is the reason why I did not use a background image and opted to use an `` tag instead

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Shisui6, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Good readme file. :heavy_check_mark: 
-   Correct GitHu Flow. :heavy_check_mark: 
-   Meaning commit messages. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**